sonar-csharp-plugin/src/main/resources/documentation.md

```diff
@@ -7,6 +7,13 @@ key: csharp
 <!-- update_center:csharp -->
 <!-- /static -->
 
+## Supported Versions
```

Comment:
As a concept, isn't this a pointless change given the fact that documentation lives only in Documentation team CMS and there's no more synchronization in place?

On the long run, these files should get dropped from all analyzers.

Reply:
Pavel, that's a good point, which we also discussed in the small circle with the PM and documentation team. We came to the conclusion that the embedded documentation is a good source of truth not only for the supported version but also for other content that can be extracted there by the documentation team. By being part of the Git history, it is also easy to keep track of how the language version support has evolved.

Reply:
The point is that these files will get dropped sooner or later in any case. So there will be no history.

Reply:
Do you have any source for the statement? The documentation team is about to discover new ways to implement automation again.

Reply:
Just the combination of facts that this is the old way of doing this and the ultimate source of documentation truth lives in the new documentation system maintained by the docs team.
So trying to automate this again sounds like a step backwards that doesn't make much sense now.